docs/site/migration/tables/lb3-only-commands.html

```diff
@@ -0,0 +1,104 @@
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb property</code>
+      </td>
+      <td>
+        Adds properties to an existing model
+      </td>
+      <td> Open the model file, and manually add a new property.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb acl</code>
+      </td>
+      <td>
+        Adds acl settings to models
+      </td>
+      <td> Use the <a href="https://loopback.io/doc/en/lb4/Loopback-component-authentication.html">AuthenticationComponent</a> and <a href="https://loopback.io/doc/en/lb4/Loopback-component-authorization.html">AuthorizationComponent</a> to set up authentication and authorization manually.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb remote-method</code>
+      </td>
+      <td>
+        Adds a new remote method to a model
+      </td>
+      <td> Open the model's controller file and add a new method manually. Depending on the purpose of this new controller method, you may also need to a new method to the model's repository.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb middleware</code>
+      </td>
+      <td>
+        Add new middleware to the application
+      </td>
+      <td> There is <a href="https://github.com/strongloop/loopback-next/pull/5118">work in progress</a> to enable express middleware in the form of interceptors that you will need to create and register
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb boot-script</code>
+      </td>
+      <td>
+        Creates a skeleton boot script in server/boot directory. Allows for customized intialization of anything during boot phase
+      </td>
+      <td> Create a <a href="https://loopback.io/doc/en/lb4/Life-cycle.html#overview">lifecycle observer</a> to run custom code during different phases of the application's lifecycle; including startup.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb export-api-def</code>
+      </td>
+      <td>
+        Export the application's API definitions to a YAML or JSON file
+      </td>
+      <td> Visit the application's /openapi.json endpoint to access your OpenAPI specification in JSON format, or the /openapi.yaml endpoint for YAML. Alternatively, the OpenAPI specification file can also be accessed in code through the getApiSpec() function from your RestServer instance. Read <a href="https://loopback.io/doc/en/lb4/Defining-the-API-using-code-first-approach.html#reviewing-your-api-specification">Reviewing your API Specification</a> for more details.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        <code>lb soap</code>
+      </td>
+      <td>
+        Create a SOAP client based on a SOAP service's WSDL file
+      </td>
+      <td> Manually create an application that calls the SOAP service through the loopback-connector-soap connector.
```

Comment:
I'm not sure if we have an equivalent or workaround for that. @raymondfeng @bajtos might be the better person to comment on that.

Reply:
Right now, there is no equivalent to `lb soap` (from LB3) in LB4. And all the documentation and code samples for interacting with web services are written for LB3. So, currently, I cannot even point users to a LB4 code snippet for any soap related stuff. This is why I mentioned this a few weeks ago in an estimation meeting. @bajtos , @raymondfeng , perhaps we can mention a backlog task url here in the meantime; until we have an equivalent command or LB4 code snippets documented?

Reply:
I think the text in the table might refer to https://loopback.io/doc/en/lb4/Calling-other-APIs-and-web-services.html, which is not quite the same as `lb soap`. This link shows the steps on how to connect to a SOAP service but it does not generate the application based on a WSDL. Perhaps before we have such functionality, we can mention this?